src/console_cmds.cpp

```diff
@@ -160,6 +160,26 @@ DEF_CONSOLE_HOOK(ConHookNoNetwork)
 	return CHR_ALLOW;
 }
 
+/**
+ * Check whether we are in singleplayer mode or are the server.
```

Comment:
The name of the hook and content of the comment do not match with the implementation. It does not check whether its the server or not in network, it also checks for being in the menu.
I don't really have an useful alternative name for the hook though, unless making it super explicit like "ConHookServerOrSinglePlayerOrEditor".

Reply:
Nevermind, found a clean solution :)